src/components/Contentful/HomePage/NewHomeLoansByCategoryGrid.vue

```diff
@@ -55,7 +55,7 @@ const loanCategoryPrefetchQuery = gql`
 
 export default {
 	name: 'NewHomeLoansByCategoryGrid',
-	inject: ['apollo', 'cookieStore'],
+	inject: ['apollo', 'cookieStore', 'device'],
```

Comment:
Just out of curiosity, where is this `device` being defined?

Reply:
check vue-middleware